src/Analyzer/Passes/QueryAnalysisPass.cpp

```diff
@@ -1717,7 +1743,8 @@ void QueryAnalyzer::evaluateScalarSubqueryIfNeeded(QueryTreeNodePtr & node, size
         if (!type->isNullable())
         {
             if (!type->canBeInsideNullable())
-                throw Exception(ErrorCodes::INCORRECT_RESULT_OF_SCALAR_SUBQUERY,
+                throw Exception(
```

Comment:
This does not look valid or better readable than before. Consider to revert this line.

Reply:
Unintentional change, reverted it